examples/design-system-tutorial/src/components/card/card.styles.ts

```diff
@@ -0,0 +1,77 @@
+import {
+    controlCornerRadius,
+    designUnit,
+    fillColor,
+    neutralForegroundRest,
+    typeRampMinus1FontSize,
+    typeRampMinus1LineHeight,
+    typeRampPlus1FontSize,
+    typeRampPlus1LineHeight,
+    elevation,
+} from "@fluentui/web-components";
+import { css } from "@microsoft/fast-element";
+import { DesignToken, display } from "@microsoft/fast-foundation";
+
+/**
+ * Not only can we have system-wide design tokens, but we can
+ * have component-specific tokens as well. This token enables
+ * changing the corner radius of cards independently of other controls.
+ * However, by default it aliases to the controlCornerRadius token.
+ */
+export const cardCornerRadius = DesignToken.create<number>(
+    "card-corner-radius"
+).withDefault(controlCornerRadius);
+
+export const demoCardStyles = css`
+    ${display("grid")} :host {
+        --elevation: 4;
+        grid-template-rows: calc(${designUnit} * 39px) 1fr auto calc(${designUnit} * 2px);
```

Comment:
I generally recommend avoiding magic numbers in calcs, and for using the `px` for the unit only (so ` * 1px`). I can let it slide here, but I think the height of that first row is aligned with the `max-height` of the slotted `heading` below: `calc(${typeRampPlus1LineHeight} * 2)`.

Reply:
Going to leave this as-is in the interest of not re-working this too much